docs/06-releasing.md

```diff
@@ -5,21 +5,38 @@
 The following steps are needed to release the app:
 
 * Tag latest git commit on master
-* Create a release in [Sentry](https://sentry.integreat-app.de) and [upload source maps](05-error-reporting.md).
+* Create a release in [Sentry](https://sentry.integreat-app.de) by [uploading source maps](05-error-reporting.md).
 * Create a release in Jira so we can reference to this later.
 * Create an entry in [CHANGELOG.md](../CHANGELOG.md)
 * Decide whether we want to send the app to testing prior to releasing
 
 ### Android
+For building the Release Android App Bundle, you need the corresponding keystore file:
+For further information regarding the keystore have a look at the `keystore_integreat.txt` in the drive.
+You need to place the keystore file in the `android/app` folder.
 
 * Increment the `versionCode` in the [build.gradle](../android/app/build.gradle)
-* Build release APK using the commands used in the [Jenksinsfile](../Jenkinsfile)
-    * For example with necessary keystore file and keys: `./gradlew -PMYAPP_RELEASE_STORE_PASSWORD=*** -PMYAPP_RELEASE_KEY_ALIAS=*** -PMYAPP_RELEASE_KEY_PASSWORD=*** -PMYAPP_RELEASE_STORE_FILE=***.jks  build -x lint -x lintVitalRelease`
-    * For further information regarding the keystore have a look at the `keystore_integreat.txt` in the drive
-* Upload APK in [Developer Console](https://play.google.com/apps/publish/)
+* For building the Release Android App Bundle use the following command:
+  ```bash
+    ./gradlew -PMYAPP_RELEASE_STORE_FILE=<store_file> -PMYAPP_RELEASE_STORE_PASSWORD=<store_pw> -PMYAPP_RELEASE_KEY_ALIAS=<key_alias> -PMYAPP_RELEASE_KEY_PASSWORD=<key_pw> bundleRelease -x lint -x lintVitalRelease
```

Comment:
Probably -x lintvitalRelease is no longer needed.

Reply:
Still failed for me without `-x lintVitalRelease`. However, I added `checkReleaseBuilds false` to build.gradle as suggested in the error message, so it should be fine now.